napari/_qt/qt_main_window.py

```diff
@@ -138,14 +138,17 @@ def current_viewer(cls):
         window = cls.current()
         return window._qt_viewer.viewer if window else None
 
-    def event(self, e):
+    def event(self, e) -> bool:
         if (
             e.type() == QEvent.ToolTip
             and self._qt_viewer.viewer.tooltip.visible
         ):
-            QToolTip.showText(
-                e.globalPos(), self._qt_viewer.viewer.tooltip.text, self
+            pnt = (
```

Comment:
Does this change catch both the name/type change from Qt5 to Qt6, but also because only some events have positions (e.g. `QMouseEvent`)? Is `QMainWindow.event` guaranteed to only get `QMouseEvents`? Regardless, a short explanatory comment here would be useful to explain why we need this check and that we could remove it if we ever deprecated Qt5 (unlikely...).